.eslintignore

```diff
@@ -15,6 +15,8 @@
 !.*
 .*/
 .eslintcache
+/.yalc*
+/yalc.lock
```

Comment:
Can revert once https://github.com/emberjs/ember-test-helpers/pull/1319 is released